internal/operators/ocs/custom_manifest.go

```diff
@@ -0,0 +1,82 @@
+package ocs
+
+import (
+	"bytes"
+	"os"
+	"text/template"
+)
+
+// Subscription Manifest for testing internal builds
+func ocsCustomSubscription() (string, error) {
+	data := map[string]string{
+		"OPERATOR_NAMESPACE":         Operator.Namespace,
+		"OPERATOR_SUBSCRIPTION_NAME": Operator.SubscriptionName,
+	}
+
+	const ocsSubscription = `apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: "{{.OPERATOR_SUBSCRIPTION_NAME}}"
+  namespace: "{{.OPERATOR_NAMESPACE}}"
+spec:
+  installPlanApproval: Automatic
+  name: ocs-operator
+  source: ocs-catalogsource
+  sourceNamespace: openshift-marketplace`
+
+	tmpl, err := template.New("ocsSubscription").Parse(ocsSubscription)
+	if err != nil {
+		return "", err
+	}
+	buf := &bytes.Buffer{}
+	err = tmpl.Execute(buf, data)
+	if err != nil {
+		return "", err
+	}
+	return buf.String(), nil
+}
+
+// CatalogSource Manifest for testing internal builds
+func ocsCatalogSource() (string, error) {
+
+	image := "quay.io/rhceph-dev/ocs-registry:latest"
```

Comment:
Actually as engineering building also previous versions for z-streams there is no latest tag.
We have tags like:
`latest-stable-4.8`
`latest-stable-4.7`
So I would go with one of this as default one. Or we have also `latest-4.8` but this build was not tested and we don't know if it's even deployable, so the stable tag is better.

Reply:
updated